packages/upload-api/test/storage/agent-store-tests.js

```diff
@@ -0,0 +1,135 @@
+import * as API from '../../src/types.js'
+
+import { sha256 } from 'multiformats/hashes/sha2'
+import * as BlobCapabilities from '@web3-storage/capabilities/blob'
+import { Console } from '@web3-storage/capabilities'
+
+import { alice, registerSpace } from '../util.js'
+
+/**
+ * @type {API.Tests}
+ */
+export const test = {
```

Comment:
This MUST test the special case for `ucan/conclude` or implementations could not adhere with the expectations we have for the blob flow to work

Reply:
Not sure I understand can you please help me better understand what special case are you talking about ? Also could we add this as a followup PR so I don't have to keep rebasing it all the time. I don't think I have introduced any new edge cases that before, but maybe I'm missing something

Reply:
Added tests to ensure that receipts inside `ucan/conclude` get indexed.